### PR TITLE
Fixed Vector.clamp_magnitude() parameters

### PR DESCRIPTION
**Pull request by novialriptide**
This PR fixes the parameter order that my `.clamp_magnitude()` used. Previously, it was `.clamp_magnitude(max, min=0)`. But now it's `.clamp_magnitude(max)` and `.clamp_magnitude(min, max)` to be more consistent with Python's `range()`.

### CODE REVIEW

**Review by MyreMylar — APPROVED** [review]
LGTM 👍

**Comment by Starbuck5** [comment]
Should this have a change to the typing?

**Comment by novialriptide** [comment]
> Should this have a change to the typing?

Oh, I forgot about that.

File: buildconfig/stubs/pygame/math.pyi [hunk 1]
```diff
@@ -197,8 +197,14 @@ class Vector2(Collection[float]):
     def move_towards_ip(
         self, target: _SupportsVector2, max_distance: float
     ) -> None: ...
-    def clamp_magnitude(self, max_length: float, min_length: float = 0) -> Vector2: ...
-    def clamp_magnitude_ip(self, max_length: float, min_length: float = 0) -> None: ...
+    @overload
+    def clamp_magnitude(self, max_length: float) -> Vector2: ...
+    @overload
+    def clamp_magnitude(self, min_length: float, max_length) -> Vector2: ...
```
Comment (ankith26) [inline buildconfig/stubs/pygame/math.pyi:203]:
```suggestion
    def clamp_magnitude(self, min_length: float, max_length: float) -> Vector2: ...
```

File: buildconfig/stubs/pygame/math.pyi [hunk 2]
```diff
@@ -338,8 +344,14 @@ class Vector3(Collection[float]):
         self, target: _SupportsVector3, max_distance: float
     ) -> None: ...
     def project(self, other: _SupportsVector3) -> Vector3: ...
-    def clamp_magnitude(self, max_length: float, min_length: float = 0) -> Vector3: ...
-    def clamp_magnitude_ip(self, max_length: float, min_length: float = 0) -> None: ...
+    @overload
+    def clamp_magnitude(self, max_length: float) -> Vector3: ...
+    @overload
+    def clamp_magnitude(self, min_length: float, max_length) -> Vector3: ...
```
Comment (ankith26) [inline buildconfig/stubs/pygame/math.pyi:350]:
```suggestion
    def clamp_magnitude(self, min_length: float, max_length: float) -> Vector3: ...
```

**Review by ankith26 — COMMENTED** [review]
Left two reviews on typehints. Additionally this also needs a round of `python3 setup.py format`

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR! 👍 🎉 
Merging this PR since the lint fail ATM is unrelated to this PR

**Pull request merged**